tools/ack-discover/service.py

```diff
@@ -0,0 +1,112 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Just a thought - you could also determine a list of all services by parsing the list of repositories in the `aws-controllers-k8s` GH organisation for `"*-controller"`?

Reply:
That is exactly what I do to get the list of ACK controllers (see the controller.py file). But to get the list of all AWS services (which may not have any ACK controller yet), I needed to take this approach.

Reply:
Oh so this CLI is meant to run against every single AWS service? That's going to be a chunky output haha